rakelib/utils.rake

```diff
@@ -68,6 +68,7 @@ class Utils
 
   # select the xcode version we want/support
   def self.version_select
+    Rake.sh 'mdimport /Applications/Xcode* && sleep 1'
```

Comment:
I agree that is far from ideal, but [this](https://circleci.com/gh/SwiftGen/StencilSwiftKit/128) run failed `rake xcode:test` yet passed `rake spm:test`, indicating that it might have a delay between indexing and get it available to `mdfind`.

Another approach might be run it before, in `lint:install` for example.

Any ideas?